noir-projects/aztec-nr/aztec/src/state_vars/private_set.nr

```diff
@@ -11,16 +11,15 @@ use crate::state_vars::storage::Storage;
 // docs:start:struct
 struct PrivateSet<Note, Context> {
     context: Context,
-    storage_slot: Field,
+    storage_slot: Point,
 }
 // docs:end:struct
 
 impl<T, Context> Storage<T> for PrivateSet<T, Context> {}
 
 impl<Note, Context> PrivateSet<Note, Context> {
     // docs:start:new
-    pub fn new(context: Context, storage_slot: Field) -> Self {
-        assert(storage_slot != 0, "Storage slot 0 not allowed. Storage slots must start from 1.");
```

Comment:
This might be a good place to implement something like #7515, if we go down that path.

Reply:
Linked there the issue in https://github.com/AztecProtocol/aztec-packages/pull/7437/commits/2c0318a1d4dca7362b77f3885eddeb364860cbea 👍